### ISSUE TITLE
Redux Plugin "except" not filtering actions

### ISSUE THREAD

**Issue by aklinker1** [infinitered/reactotron#1082]
I'm using `react-native-offline` and want to exclude the actions that are logged from it. I have tried adding the action type to the `except` option for `reactotron-redux`, but it is not working.

```ts
// src/utils/setup/Reactotron.ts
import Reactotron from 'reactotron-react-native';
import { reactotronRedux } from 'reactotron-redux';

export default Reactotron.configure().useReactNative({
    asyncStorage: false,
    editor: true,
    networking: {
      ignoreUrls: /google\.com/, // Ignore the connection PINGs from react-native-offline, this works
    },
    overlay: false,
  })
  .use(reactotronRedux({
    except: ['@@network-connectivity/CONNECTION_CHANGE'], // This doesn't work
  }))
  .connect();
```

and

```ts
// src/utils/setup/Store.ts
import Reactotron from './Reactotron';

// ...
enhancers = compose(
  applyMiddleware(...middleware),
  Reactotron.createEnhancer(),
);
const store = createStore(createRootReducer(), enhancers);
```

All log statements use the default export from `src/utils/setup/Reactotron`, not the one from `reactotron-react-native`.

I also tried changing the `except`'s first item to a function after I looked at the [source code](https://github.com/infinitered/reactotron-redux/blob/7826f518dccd3b6e3cbc3dfc0503e4b0ad107cab/src/customDispatch.ts#L25) for custom dispatches, and logging something inside there, but the function was never called. This makes me think that the plugin options are not passed to where they need to be, especially because [this function](https://github.com/infinitered/reactotron-redux/blob/7826f518dccd3b6e3cbc3dfc0503e4b0ad107cab/src/index.ts#L10) does not do anything with the `except` field, it just gets stripped when creating the `mergedPluginConfig` variable.

**Comment by rmevans9** [comment]
This sounds like a problem. I will be looking into it.

**Comment by rmevans9** [comment]
This is fixed in `reactotron-redux` version 3.1.1

**Comment by aklinker1** [comment]
Yep, that fixed the problem. Thanks!
